recipes/libs/libtasn1.yaml

```diff
@@ -0,0 +1,21 @@
+inherit: [autotools]
+
+metaEnvironment:
+    PKG_VERSION: "4.19.0"
+    PKG_LICENSE: "GPL-3.0+ (tests, tools), LGPL-2.1+ (library)"
```

Comment:
Again, if this creates executables, they should be split off into dedicated multiPackages.